### ISSUE TITLE
v25.0.0 breaks linux AppImage build

### ISSUE THREAD

**Issue by arantes555** [electron-userland/electron-builder#8351]
* **Electron-Builder Version**: `25.0.0`

* **Node Version**: tried multiple versions

* **Electron Version**: `31.2.1`
* **Electron Type (current, beta, nightly)**: current

* **Target**: Linux AppImage

When building with electron-builder `25.0.0`, the resulting AppImage fails to launch on Debian 12, with the following error:
```
Squashfs image uses (null) compression, this version supports only xz, zlib.
Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage
if you run it with the --appimage-extract option.
See https://github.com/AppImage/AppImageKit/wiki/FUSE
for more information
open dir error: No such file or directory
```

When rolling-back electron-builder to `24.13.3`, the built AppImage launches without problem.

This seems related to https://github.com/electron-userland/electron-builder/issues/6678 or https://github.com/electron-userland/electron-builder/issues/6624.

**Comment by mmaietta** [comment]
Thanks for the report!

@beyondkmp I think we may need to downgrade app-builder-bin to use AppImage 12.0.0 since this error is due to 13.0.0
https://github.com/develar/app-builder/pull/113

What are your thoughts?

**Comment by mmaietta** [comment]
Found an alternative route using current mksquashfs version, just seems to need this PR to resolve https://github.com/develar/app-builder/pull/123

Tested locally using `CUSTOM_APP_BUILDER_PATH=/Users/mmaietta/Development/app-builder/mac/app-builder_arm64 yarn app:pack -- -l` to generate an AppImage with the app-builder changes locally

**Comment by mmaietta** [comment]
Alrighty, the fix should be in 25.0.1 once it deploys!

**Comment by arantes555** [comment]
@mmaietta Sorry but this seems to be still broken in 25.0.2 :/ I still get the same error message.

**Comment by mmaietta** [comment]
@beyondkmp would you be willing to take a look at this? I was unable to reproduce on my ParallelsVM with AppImage builds during my local testing

**Comment by mmaietta** [comment]
@arantes555 can you share a minimum reproducible repo? Or at least your electron-builder configuration? Trying to discern how to repro this locally

**Comment by arantes555** [comment]
@mmaietta here is a repo with a reproduction : https://github.com/arantes555/electron-builder-issue8351

I based it on our project and cut most of everything, sorry if I didn't make it 100% as minimal as possible.

**Comment by mmaietta** [comment]
Alright, I've pushed a fix but am waiting on another PR to (hopefully) get included today/tomorrow in this next patch release. Then will redeploy a new release.

**Comment by mmaietta** [comment]
Please try 25.0.3. This should be the same appimage binary being used in app-builder-bin 4.0

**Comment by arantes555** [comment]
@mmaietta Yep, seems to work ! Thank you ! Closing this now :)
